framework/validators/RangeValidator.php

```diff
@@ -23,7 +23,15 @@
 class RangeValidator extends Validator
 {
     /**
-     * @var array list of valid values that the attribute value should be among
+     * @var mixed list of valid values that the attribute value should be among or a PHP callable that returns
```

Comment:
It was just copied from here:
https://github.com/yiisoft/yii2/blob/master/framework/validators/DefaultValueValidator.php#L22